### PR TITLE
Added new bundlesize-pipe executable, which accepts file contents fro…

### LINKED ISSUE #90: [feature request] Accept piped file
**Why?**
Would be super useful to have this in place so we could pipe in output of another program, i.e. one which strips down comments.

Im happy to help with implementing this, just reaching out first with the idea as suggested in CONTRIBUTING section ;)

### PR DESCRIPTION
**Pull request by Andarist**
…m the stdin stream (fixes #90)

### CODE REVIEW

File: pipe.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const reporter = require('./src/reporter')
+const gzip = require('gzip-size')
+const bytes = require('bytes')
+const readStream = require('./src/readStream')
+const { error } = require('prettycli')
+const build = require('./src/build')
+const debug = require('./src/debug')
+
+if (Boolean(process.stdin.isTTY)) {
+  error('bundlesize-pipe executable is meant for usage with piped data.', { label: 'ERROR', exit: true })
```
Comment (Andarist) [inline pipe.js]: option obejct could be omitted if this gets released - https://github.com/siddharthkp/prettycli/pull/3/files

File: src/files.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ config.map(file => {
   } else {
     paths.map(path => {
       const size = gzip.sync(fs.readFileSync(path, 'utf8'))
-      const maxSize = bytes(file.threshold || file.maxSize) || Infinity
+      const maxSize = bytes(file.maxSize) || Infinity
```
Comment (Andarist) [inline src/files.js]: i have not found how this `.threshold` could land here, its probably some deprecated (and removed) option

**Comment by jagzviruz** [comment]
Can you resolve package-lock.json and raise a fresh pull ?

**Comment by Andarist** [comment]
@jagzviruz Ive rebased this one instead. So its quite fresh right now :)

**Comment by SaraVieira** [comment]
@Andarist You have one lint error 😟

/home/travis/build/siddharthkp/bundlesize/src/readStream.js
  1:20  error  Missing space before function parentheses  space-before-function-paren

**Comment by Andarist** [comment]
@SaraVieira fixed, i believe this was introduced by some new rules which were not there when Ive created the PR

**Comment by SaraVieira** [comment]
@Andarist Very likely , I have updated a bunch of eslint rules a couple of weeks ago

**Comment by Andarist** [comment]
@SaraVieira @reznord thanks for merging :) whats ur release cycle, when I can expect to have this published? No rush ofc, just asking

**Comment by SaraVieira** [comment]
If everyone os okay with it I can do a minor release of this today☺️

@reznord @jagzviruz

**Comment by palashmon** [comment]
Yup, go ahead!

Once a version is released, just tag me here once so that I can test if the install is working fine and there is no issue like #130

**Comment by Andarist** [comment]
@SaraVieira just pinging about the release :P

**Comment by SaraVieira** [comment]
@Andarist omg I am so sorry , I have been doing a 100 stuff and forgot , if you ever meet me in person I do owe you coffee or beer :(

Published:
https://github.com/siddharthkp/bundlesize/releases/tag/v0.15.0
https://www.npmjs.com/package/bundlesize

**Comment by Andarist** [comment]
@SaraVieira not a problem, i totally understand how it is to take too much on yourself

**Comment by SaraVieira** [comment]
Still if you ever in lisbon ping me and I shall get you coffee 🙌

Thank you for the ping and the understanding 😄

**Pull request merged**